src/pages/WatchVideo/WatchVideo.jsx

```diff
@@ -0,0 +1,44 @@
+// import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { embeddedLink } from '../../utils/video-details-services';
+// import { videos } from '../../backend/db/videos';
+// import { useHistory } from '../../context';
+
+const WatchVideo = () => {
+    const { videoId } = useParams();
+    // const { setHistoryHandler } = useHistory();
+
+    // useEffect(() => {
+    //     let vid;
```

Comment:
you can remove unrequired comments

Reply:
Will do, this part isn't complete yet. I was just using it to test something.